vscode.md

```diff
@@ -0,0 +1,53 @@
+# express-server
+
+ # Features of VS Code
```

Comment:
- remove extra space prior to the start of the heading

- make this as h2 tag

Reply:
Done